tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalServerBehavior.java

```diff
@@ -234,6 +240,39 @@ private PortalServer getPortalServer()
 
     private String[] getRuntimeStartVMArguments()
     {
+        if( getPortalServer().getDeveloperMode() )
+        {
+            File portalext = getPortalRuntime().getLiferayHome().append( "portal-ext.properties" ).toFile();
+
+            if( !portalext.exists() )
+            {
+                try
+                {
+                    portalext.createNewFile();
+                }
+                catch( IOException e )
+                {
+                }
+            }
+
+            try(InputStream in = new FileInputStream( portalext ); OutputStream out = new FileOutputStream( portalext ))
+            {
+                Properties properties = new Properties();
```

Comment:
Use this class instead `com.liferay.ide.core.properties.PortalPropertiesConfiguration` so that we save whitespace, etc.